client/view-v2.js

```diff
@@ -2305,44 +2305,46 @@ module.exports = (function () {
     })
 
     // content fields
-    Object.entries(contentFields).forEach(([contentFieldName, contentFieldValue]) => {
-      if (
-        (formData?.[contentFieldName] === undefined ||
-          formData?.[contentFieldName]?.delete === true) &&
-        noteObj?.content?.[contentFieldName]?.value === undefined
-      ) {
-        // do not return field
-        return
-      }
-
-      var valueObj = contentFieldValue.value
-      if (valueObj) {
+    if (contentFields) {
```

Comment:
instead of adding a new if, it isn't easier to do `Object.entries(contentFields || [])` ?